### PR TITLE
[MM-46944] : Tighten up spaces between categories in the LHS

### PR DESCRIPTION
**Pull request by gitstart**
Summary
Tighten up spaces between categories in the LHS

Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/21202
Jira: https://mattermost.atlassian.net/browse/MM-46944
Screenshots
Before:
Screenshot 2023-01-10 at 21 09 38

After:
Screenshot 2023-01-10 at 09 49 51

### Release note
```release-note
Fixed spaces between the  categories and maintain same spacing in the LHS
```

### CODE REVIEW

File: sass/layout/_sidebar-left.scss [hunk 1]
```diff
@@ -963,11 +963,7 @@ $sidebarOpacityAnimationDuration: 0.15s;
     /* Content */
     .SidebarChannelGroup_content {
         min-height: 2px;
-        margin-bottom: 14px;
-
-        &.hasFollowingSibling {
```
Comment (AshishDhama) [inline sass/layout/_sidebar-left.scss:968]: Could you please remove this https://github.com/mattermost/mattermost-webapp/blob/022ced2c95214fc225b24e2f9b0ceccbdd42f7e4/components/sidebar/sidebar_category/sidebar_category.tsx#L411

**Review by michelengelen — APPROVED** [review]

File: sass/layout/_sidebar-left.scss (hunk 1 above)
Comment (gitstart) [reply sass/layout/_sidebar-left.scss:968]: Addressed

**Review by AshishDhama — APPROVED** [review]

**Comment by matthewbirtch** [comment]
@esethna and @jgilliam17 this is ready for your review

**Review by jgilliam17 — APPROVED** [review]
Thanks @gitstart 
Tested, looks good to merge.
Verified spacing between categories on the sidebar.
[E2E report ](https://automation-dashboard.vercel.app/cycle/291593)- no PR related failures.

**Comment by jgilliam17** [comment]
Apologies, I forgot to check global drafts. 
I noticed there is an extra gap between Threads and global drafts.
Screen Shot 2023-01-27 at 4 40 01 PM

**Comment by matthewbirtch** [comment]
> Apologies, I forgot to check global drafts. I noticed there is an extra gap between Threads and global drafts. Screen Shot 2023-01-27 at 4 40 01 PM

Good catch @jgilliam17. Drafts should be grouped together with Insights and Threads and you're right - it shouldn't have this extra space

**Review by jgilliam17 — CHANGES_REQUESTED** [review]
Thanks for fixing drafts related gap issue @gitstart 
I have one more request
When moving categories using grab and drop, I am noticing a gap between hand pointer and category being moved. See video.
Expected -  hand pointer should remain on top of the category or channel that is being moved
Observed - category/channel hangs below pointer

https://user-images.githubusercontent.com/52937121/215849456-4f456631-a9b9-49b2-a5b0-f7b8a85230bd.mov

Compared to current implementation

https://user-images.githubusercontent.com/52937121/215849571-7ab22add-b3f8-411c-82e5-85c406b9a527.mov

**Comment by gitstart** [comment]
> Thanks for fixing drafts related gap issue @gitstart I have one more request When moving categories using grab and drop, I am noticing a gap between hand pointer and category being moved. See video. Expected - hand pointer should remain on top of the category or channel that is being moved Observed - category/channel hangs below pointer
> 
>  Screen.Recording.2023-01-31.at.1.20.33.PM.mov 
> Compared to current implementation
> 
>  Screen.Recording.2023-01-31.at.1.24.00.PM.mov

@jgilliam17 fixed the pointer 
Screen Recording 2023-02-03 at 12.27.56.mov

**Review by stevemudie — APPROVED** [review]
The changes Jelena requested are in place and worked as expected.

This is all set. Looks good!

**Pull request merged**